### PR TITLE
DOC: update README.rst

### PR DESCRIPTION
**Pull request by dmgav**
Update `README.rst` to reflect latest changes. The changes include:

- Plan queue items are referenced as `items` instead of `plans` wherever appropriate.

- Included example for inserting `queue_stop` instruction to the plan queue.

- Document structure was improved.

### CODE REVIEW

**Review by mrakitin — COMMENTED** [review]
Didn't I submit my review? :(

File: README.rst [hunk 1]
```diff
@@ -165,7 +183,12 @@ Push a new plan to the back of the queue::
 It takes 10 second to execute the third plan in the group above, so it is may be the most convenient for testing
 pausing/resuming/stopping of experimental plans.
 
-The plan to be added at any position of the queue including pushing to the front or back of the queue::
+API for queue operations are designed work identically with items of all types. For example, a 'queue_stop`
```
[the remark this answers is not in the document]
Comment (dmgav) [reply README.rst:186]: Changes are implemented in https://github.com/bluesky/bluesky-queueserver/pull/98

File: README.rst [hunk 2]
```diff
@@ -251,10 +274,10 @@ Plans can be read from the queue without changing it. `queue_item_get` requests
   http POST http://localhost:60610/queue/item/get pos:=-2
   http POST http://localhost:60610/queue/item/get uid:='<uid>'
 
-Plans can be moved within the queue. Plans can be addressed by position or UID. If plans are
-addressed by position, then the plan is moved from 'source' position to 'destination' position.
-If plans are addressed by UID, then the plan with <uid_source> in inserted before or after
-the plan with <uid_dest>::
+Items can be moved within the queue. Items can be addressed by position or UID. If positional addressing
+is used then items are moved from 'source' position to 'destination' position.
+If items are addressed by UID, then the item with <uid_source> in inserted before or after
```
[the remark this answers is not in the document]
Comment (dmgav) [reply README.rst:279]: Changes are implemented in https://github.com/bluesky/bluesky-queueserver/pull/98

[the remark this answers is not in the document]
Comment (mrakitin) [reply README.rst:279]: Thank you.

**Pull request merged**